internal/str/str.go

```diff
@@ -59,3 +59,22 @@ func Colorize(text string) string {
 	}
 	return text
 }
+
+func BytesToMegabytes(bytesStr string) (string, error) {
+	bytes, err := strconv.ParseFloat(bytesStr, 64)
+	if err != nil {
+		return "", err
+	}
+	mb := bytes / (1024.0 * 1024.0)
+	return fmt.Sprintf("%.2f MBs", mb), nil
+}
+
+func MsToSecs(ms string) (string, error) {
+	milliseconds, err := strconv.ParseInt(ms, 10, 64)
+	if err != nil {
+		return "", err
+	}
+	seconds := float64(milliseconds) / 1000.0
+	secondsStr := fmt.Sprintf("%.1f sec", seconds)
+	return secondsStr, nil
+}
```

Comment:
These functions are used only in `estimate_stages.go` and they are not very generic.
So, it would be better to unexport them and move to `estimate_stages.go`

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/422/commits/0433098cea6f8d20645c6af77b458abd55f76fcc

Reply:
Could you also unexport them?

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/422/commits/1cd2f8c2fc7d42b74fc743c48b025caede9c0f06